core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -228,6 +239,26 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
     verifyAlterBrokerConfig(node, "1", List("--entity-name", "1"))
   }
 
+  @Test
+  def shouldAddBrokerLoggerConfig(): Unit = {
+    val node = new Node(1, "localhost", 9092)
+    verifyAlterBrokerLoggerConfig(node, "1", "1", List(
+      new ConfigEntry("kafka.log.LogCleaner", "INFO"),
+      new ConfigEntry("kafka.server.ReplicaManager", "INFO"),
+      new ConfigEntry("kafka.server.KafkaApi", "INFO")
+    ))
+  }
+
+  @Test(expected = classOf[InvalidConfigurationException])
+  def shouldRaiseInvalidConfigurationExceptionWhenAddingInvalidBrokerLoggerConfig(): Unit = {
+    val node = new Node(1, "localhost", 9092)
+    // we want to alter kafka.log.LogCleaner, kafka.server.ReplicaManager and kafka.server.KafkaApi
+    // yet, we only get one logger returned
```

Comment:
I don't think I understand this comment or this test... what is invalid in this config? If we expect an exception, why does the comment mention only one logger returned?

Reply:
I clarified the comment better:
```
// verifyAlterBrokerLoggerConfig tries to alter kafka.log.LogCleaner, kafka.server.ReplicaManager and kafka.server.KafkaApi
    // yet, we make it so DescribeConfigs returns only one logger, implying that kafka.server.ReplicaManager and kafka.log.LogCleaner are invalid
```

it's kind of tied to the implementation of `verifyAlterBrokerLoggerConfig`

Reply:
if it doesn't throw an exception, the name is pretty misleading. maybe "shouldNotReturnLoggerWhenAddingInvalidLogger"?